source/topic3.rst

```diff
@@ -208,16 +208,232 @@ Getters and Setters
 toString
 ^^^^^^^^
 
+* You may remember that ``__repr__`` in python was a special method that would return a string version of the object
+    * And, if you called ``print(some_object)``, the ``__repr__`` would automatically get called
+
+* When we wrote our own classes, we ended up overriding the ``__repr__`` we got for free
+    * The default one simply prints out the object name and a memory address, which wasn't very helpful
+    * ``<__main__.Friend object at 0x7f130d9c52e0>``
+    * *Technically*, the behaviour we get for free is inherited from the closest parent class
```

Comment:
Maybe parent -> super to stick to the language given terminology.

Reply:
<3